src/lib/containers/download_list_v2/DirectProgrammaticDownload.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react'
+import ReactTooltip from 'react-tooltip'
+import IconSvg from '../IconSvg'
+import { ProgrammaticInstructionsModal } from '../ProgrammaticInstructionsModal'
+import { TOOLTIP_DELAY_SHOW } from '../table/SynapseTableConstants'
+
+type DirectProgrammaticDownloadProps = {
+  entityId: string
+  version?: number
+}
+
+function DirectProgrammaticDownload({
+  entityId,
+  version
+}: DirectProgrammaticDownloadProps) {
+  const [isShowingModal, setIsShowingModal] = useState<boolean>(false)
+
+  const rCode = `library(synapser)
+synLogin('username','password')
```

Comment:
Doesn't need to be resolved in this PR, but we should modify these instructions to support Personal Access Tokens instead of a password

Reply:
Good point